irc/config.go

```diff
@@ -1778,6 +1779,12 @@ func (config *Config) generateISupport() (err error) {
 	}
 	isupport.Add("WHOX", "")
 
+	for key, value := range config.Server.AdditionalISupport {
+		if !isupport.Contains(key) {
+			isupport.Add(key, value)
```

Comment:
We should actually get improved validation here to ensure that the final token is syntactically valid:

https://modern.ircdocs.horse/#rplisupport-005

`RegenerateCachedReply` does basic checks (for an initial colon or a space) but it's not really adequate for user-supplied data.

Reply:
Nei suggests that there are no real restrictions beyond those imposed by C2S itself; let's check the total token length and check for the 3 forbidden octets.